lib/graphql/subscriptions/instrumentation.rb

```diff
@@ -44,7 +44,10 @@ def initialize(inner_proc)
 
         # Wrap the proc with subscription registration logic
         def call(obj, args, ctx)
+          @inner_proc.call(obj, args, ctx) if @inner_proc
```

Comment:
I monkey patched this change into my app, and had to make this change:

```ruby
if @inner_proc && !@inner_proc.is_a?(GraphQL::Field::Resolve::BuiltInResolve)
  @inner_proc.call(obj, args, ctx)
end
```

otherwise my subscriptions that do not have a custom resolve, would fail.

Reply:
Hmm I think you're right. It's the default behaviour for the getter. I'll update my MR & test